js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -99,6 +99,14 @@ const DropDownEditor = TextBox.inherit({
         });
     },
 
+    _isInstantlyMode() {
+        return this.option('applyValueMode') === 'instantly';
+    },
+
+    _shouldBeClosed() {
```

Comment:
Is it necessary to put it in a separate method? Why we can't to write as follows
`if(this._isInstantlyMode()){}`. In condition block we see that DDEditor will close.